### PR TITLE
tests: add tests for search api with numeric attrs

### LINKED ISSUE #739: Test numeric SEARCH queries
We can SEARCH using numeric comparisons since https://github.com/nspcc-dev/neofs-node/pull/2733 (see https://github.com/nspcc-dev/neofs-api/issues/265 also), this needs to be tested for small/big objects with positive/negative scenarios.

### PR DESCRIPTION
**Pull request by evgeniiz321**
closes #739

### CODE REVIEW

File: pytest_tests/tests/object/test_object_api.py [hunk 1]
```diff
@@ -52,6 +53,16 @@
 RANGE_MAX_LEN = 500
 # Used for static ranges found with issues
 STATIC_RANGES = {}
+OBJECT_NUMERIC_VALUES = [-1, 0, 1, 10]
```
Comment (roman-khimov) [inline pytest_tests/tests/object/test_object_api.py:56]: We need some `xyz` value as well to check that this object is never returned for numeric searches (attributes can have any values).

Some large (>2^64) values would also be nice to check.

Comment (evgeniiz321) [reply pytest_tests/tests/object/test_object_api.py]: Do you mean this filter `attr GT xyz`? if we have letters instead of numbers we got `invalid search query` error, I have a test for that. 

For large value - added it.

Comment (roman-khimov) [reply pytest_tests/tests/object/test_object_api.py]: I mean searching for `attr GT 0`, but having an object with `attr=xyz`.

File: pytest_tests/lib/helpers/grpc_responses.py [hunk 2]
```diff
@@ -40,6 +40,8 @@
     "table validation: it is prohibited to modify system access"
 )
 
+INVALID_SEARCH_QUERY = ".*invalid search query.*"
```
Comment (cthulhu-rider) [inline pytest_tests/lib/helpers/grpc_responses.py:43]: i'd also add query(s) with floating point numbers which are also currently unsupported. `attr<1.2` is incorrect as well as `attr<text` but not so obvious

File: pytest_tests/tests/object/test_object_api.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/tests/object/test_object_api.py]: added both scenarios

File: pytest_tests/lib/helpers/grpc_responses.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/lib/helpers/grpc_responses.py:43]: added this case as well

File: pytest_tests/tests/object/test_object_api.py [hunk 3]
```diff
@@ -281,6 +292,133 @@ def _generate_filters_expressions(attrib_dict: dict[str, str]):
...
+    def test_object_search_with_numeric_queries(
+        self, default_wallet: NodeWallet, container: str, object_size: int, operator: str
+    ):
+        objects = []
+        for numeric_value in OBJECT_NUMERIC_VALUES:
+            file_path = generate_file(object_size)
+
+            objects.append(
+                {
+                    "numeric_value": numeric_value,
```
Comment (cthulhu-rider) [inline pytest_tests/tests/object/test_object_api.py:310]: `"numeric_value"` key string used multiple times, can we define a const for it?

**Review by cthulhu-rider — APPROVED** [review]

File: pytest_tests/tests/object/test_object_api.py (hunk 3 above)
Comment (evgeniiz321) [reply pytest_tests/tests/object/test_object_api.py]: Thank you, there was already a const for it, i just didn't use it here. Fixed.

**Review by roman-khimov — APPROVED** [review]
